WolvenKit.App/ViewModels/Tools/ImportableItemViewModel.cs

```diff
@@ -153,12 +153,18 @@ public static bool TryLoadXbmSettingsFromGame(string fileName, IArchiveManager a
         using MemoryStream stream = new();
         file.Value.Extract(stream);
         if (!parserService.TryReadRed4File(stream, out var cr2WFile) ||
-            cr2WFile.RootChunk is not CBitmapTexture { Setup: { } setup } bitmapTexture ||
+            cr2WFile.RootChunk is not CBitmapTexture { Setup: STextureGroupSetup setup } bitmapTexture ||
             bitmapTexture.RenderTextureResource.RenderResourceBlobPC.Chunk is not rendRenderTextureBlobPC blob)
         {
             return false;
         }
 
+        // Invalid enum value. Not sure where this is coming from, in the constructors everything is assigned correctly.
+        if (setup.Group.ToString() == "0")
+        {
+            setup.Group = Enums.GpuWrapApieTextureGroup.TEXG_Generic_Color;
+        }
```

Comment:
That looks wrong. Do you have an example file?
Or a way to reproduce this?

Not sure why it is in 2 PRs...

Reply:
Just tested it with ~6000 random files (including the mentioned ones)
Not a single throw that issue for me (with your fix removed from dev)